resources/lib/kodi_monitor.py

```diff
@@ -252,7 +265,7 @@ def monitor_radiostream(self):
             # another monitoring already in progress...
             return
         last_title = ""
-        while not self.abortRequested() and getCondVisibility("Player.HasAudio"):
```

Comment:
sorry, this one should be revised because the loop will not end untill kodi is shutdown